components/script/dom/html/htmlinputelement.rs

```diff
@@ -609,9 +608,10 @@ impl HTMLInputElement {
     pub(crate) fn enable_sanitization(&self) {
         self.sanitization_flag.set(true);
         let mut textinput = self.textinput.borrow_mut();
-        let mut value = textinput.single_line_content().clone();
+        // FIXME: Don't convert utf16->utf8->utf16 here
```

Comment:
We'll easily lose track of these FIXME. Could you file an issue and change these FIXME into TODO(issue_number) when this PR is ready to merge?

Ditto bellow.